template/utils/config.py

```diff
@@ -158,7 +157,7 @@ def add_args(cls, parser):
         "--neuron.run_id",
         type=str,
         help="The DHT run_id",
-        default="s25_run_v1",
+        default="s25_run_test1",
```

Comment:
can we keep this to s25_run_v1?

Reply:
Yeah, was using it for testing, so wouldn't intefere wiht other runs